testutils/custom-params/include/testutils/CustomParameters.hpp

```diff
@@ -17,31 +17,12 @@
  * License-Filename: LICENSE
  */
 
-#include <map>
 #include <string>
-#include <vector>
 
-/**
- * Currently supported parameters and syntax
- *
- * --credentials=<s>
- * where <s> is the test application credentials in JSON format
- *
- * --runProdTests
- * when specified tests are run on production server
- * when not specified tests are run on staging server
- *
- */
+/// Class that provides access to global variables that are used during tests.
 class CustomParameters {
```

Comment:
Mark class's ctor as = delete, because it no make sense to construct it.

Reply:
In this case, it's better to have a free function, but this might affect iOS and Android test builds